### DISCUSSION TITLE
Mark folders as unwanted on 2.0.0

### DISCUSSION THREAD

**Discussion by glottisfaun0000** [VueTorrent/VueTorrent#1245] [Q&A] [answered]
Have we lost the ability to mark whole folders rather than individual files as unwanted in the Content tab, or am I missing something obvious?

**Comment by Larsluph** [comment]
Content tab do have a lot of regression, I'm planning on reworking the whole system but I'm still trying to figure out how we can make that better.

**Reply by glottisfaun0000** [reply]
Gotcha, looking forward to a new approach for content that also has some sort options. Everything else about 2.0 is awesome, thanks for working on it

**Reply by Larsluph** [reply]
Glad you like it ^^

**Comment by glottisfaun0000** [comment, marked as answer]
we're so back with 2.6.0 🎊

**Reply by glottisfaun0000** [reply]
also just wanna mention somewhere that the "active" option for torrent state filter is fantastic and i was gonna request something similar. personally, the way i use filtering, i usually want to also filter for Done torrents, which basically means I wish I could click "Seeding" twice to just negate it and view everything else, but the active option gets me 99% of the way there.
